core/src/test/java/org/elasticsearch/index/query/functionscore/FunctionScoreQueryBuilderTests.java

```diff
@@ -107,6 +110,12 @@ protected FunctionScoreQueryBuilder doCreateTestQueryBuilder() {
     }
 
     @Override
+    protected String[] shuffleProtectedFields() {
+        // do not shuffle fields that may contain arbitrary content
+        return SHUFFLE_PROTECTED_FIELDS;
```

Comment:
Had to make this change now that `XContent` is shuffled for objects inside lists.